Gemfile

```diff
@@ -10,6 +10,9 @@ gem 'moab-versioning', '>= 4.2.0' # work with Moab Objects; 4.2.0 has DepositBag
 gem 'lyber-core' # robot code
 gem 'robot-controller' # requires Resque
 gem 'honeybadger' # for error reporting / tracking / notifications
+gem 'text-table' # to generate tables for StatsReporter
+gem 'faraday' # to query workflows for StatsReporter
```

Comment:
question:  I'm assuming this means dor-workflow-services aren't adequate in this case?